index.js

```diff
@@ -425,4 +425,19 @@
     return JSON.parse(s);
   });
 
+  // Regex  //////////////////////////////////////////////////////////////////
+
+  //. match :: RegEx -> String -> Maybe [*]
```

Comment:
The type of the first argument should be written `RegExp` to match the name of the constructor function. Also, could the `*` be replaced with `String`?